driver-core/src/main/com/mongodb/internal/capi/MongoCryptHelper.java

```diff
@@ -72,9 +78,31 @@ public static MongoCryptOptions createMongoCryptOptions(final Map<String, Map<St
             spawnArgs.add("--idleShutdownTimeoutSecs");
             spawnArgs.add("60");
         }
+        if (!spawnArgs.contains("--logpath")) {
+            spawnArgs.add("--logpath");
+            spawnArgs.add(System.getProperty("os.name").startsWith("Windows") ? "NUL" : "/dev/null");
+        }
         return spawnArgs;
     }
 
-    private MongoCryptOptionsHelper(){
+    public static MongoClientSettings createMongocryptdClientSettings(final String connectionString) {
+
+        return MongoClientSettings.builder()
+                .applyToClusterSettings(new Block<ClusterSettings.Builder>() {
+                    @Override
+                    public void apply(final ClusterSettings.Builder builder) {
+                        builder.serverSelectionTimeout(1, TimeUnit.SECONDS);
+                    }
+                })
+                .applyToSocketSettings(new Block<SocketSettings.Builder>() {
+                    @Override
+                    public void apply(final SocketSettings.Builder builder) {
+                        builder.readTimeout(1, TimeUnit.SECONDS);
+                        builder.connectTimeout(1, TimeUnit.SECONDS);
+                    }
+                })
+                .applyConnectionString(new ConnectionString((connectionString != null)
```

Comment:
Small behavioral change: apply the connection string after the defaults, so that they can be overridden by the application if desired.

Reply:
Ack